### ISSUE TITLE
feat: Google Chat Integration

### ISSUE THREAD

**Issue by clanto007** [chatwoot/chatwoot#3545]
**Is your enhancement request related to a problem? Please describe.**
Google Chat can be a very useful channel to have for customers with google workspace accounts so that they can use the chat included for support.

**Describe the solution you'd like**
A bot in google chat created in gogole workspace tenant. As for telegram bot.

**Comment by JarydX** [comment]
I agree to this requests. Our users are in Google Workspace at home, a bot-channel in Google Chat to contact us with ease would bei great!

**Comment by sojan-official** [comment]
Thanks for the feedback, We will keep this in our backlog for prioritization.
We will share further updates when the team starts working on this.

**Comment by LeonAlvarez** [comment]
Any update on Google Chat integration?

**Comment by donbowman** [comment]
https://help.socialintents.com/article/217-how-to-embed-google-chat-in-your-website this shows an example of how this integration might look.

**Comment by stonecharioteer** [comment]
This issue is being closed due to lack of activity for over a year. [For more details, visit this discussion.](https://github.com/orgs/chatwoot/discussions/12810)
